### ISSUE TITLE
Cannot get nodes running on other machines

### ISSUE THREAD

**Issue by RaresAmbrus** [strands-project/strands_3d_mapping#55]
The `cloud_merge/launch/mapping.launch` starts the nodes `semantic_map`, `cloud_merge` and `scitos_ptu/ptu_action_server_metric_map.py`. It supports launching `semantic_map`, `cloud_merge` on one machine and `scitos_ptu/ptu_action_server_metric_map.py` on another through different machine and user arguments. 

However, at the moment this is not working properly, moreover it's difficult to debug what's going on since there is not console output from the nodes launched on another machine; for now I am removing starting the action server from the same file. 

To launch the metric mapping components one will have to do:

```
roslaunch cloud_merge mapping.launch
roslaunch scitos_ptu ptu_action_server_metric_map.launch
```

I will update the readme file to reflect this.

**Comment by ghost** [comment]
actually if I run the second command, it will shutdown after the
`mapping.launch` apparently restarts the action server.

**Comment by RaresAmbrus** [comment]
Yes, I've opened a pull request where I commented out starting the action server from `roslaunch cloud_merge mapping.launch`.  It's this one: https://github.com/strands-project/strands_3d_mapping/pull/56
